src/cascadia/TerminalControl/SearchBoxControl.h

```diff
@@ -28,29 +28,23 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
         void TextBoxKeyDown(winrt::Windows::Foundation::IInspectable const& /*sender*/, winrt::Windows::UI::Xaml::Input::KeyRoutedEventArgs const& e);
 
-        bool GoForward();
-        bool IsCaseSensitive();
-
         void SetFocusOnTextbox();
         bool ContainsFocus();
 
         void GoBackwardClicked(winrt::Windows::Foundation::IInspectable const& /*sender*/, winrt::Windows::UI::Xaml::RoutedEventArgs const& /*e*/);
         void GoForwardClicked(winrt::Windows::Foundation::IInspectable const& /*sender*/, winrt::Windows::UI::Xaml::RoutedEventArgs const& /*e*/);
         void CloseClick(winrt::Windows::Foundation::IInspectable const& /*sender*/, winrt::Windows::UI::Xaml::RoutedEventArgs const& e);
 
-        TYPED_EVENT(Search, TerminalControl::SearchBoxControl, winrt::hstring);
+        WINRT_CALLBACK(Search, SearchHandler);
         TYPED_EVENT(Closed, TerminalControl::SearchBoxControl, Windows::UI::Xaml::RoutedEventArgs);
 
     private:
-        bool _goForward; // The direction of the search, controlled by the buttons with arrows
-
-        winrt::Windows::UI::Xaml::Controls::Primitives::ToggleButton _goForwardButton;
-        winrt::Windows::UI::Xaml::Controls::Primitives::ToggleButton _goBackwardButton;
-        winrt::Windows::UI::Xaml::Controls::Primitives::ToggleButton _caseButton;
         winrt::Windows::UI::Xaml::Controls::TextBox _textBox;

```

Comment:
Hey if we're getting rid of the rest of these members, can we also replace `_textBox` with just `TextBox()`?

Reply:
changed